### DISCUSSION TITLE
地址仅跳转一个网址。

### DISCUSSION THREAD

**Discussion by rsecc** [travellings-link/travellings#1777] [寻求帮助] [answered]
我在我的电脑上点击开往的跳转地址：https://www.travellings.cn/go.html，两天了，都只能跳转到 https://fu1fan.cn/ 这一个网址。

**Comment by tjsky** [comment]
https://github.com/travellings-link/travellings/assets/7272911/6d302847-b180-468d-aea3-b11e99028a33

我这里则是一直只跳转到`https://www.xiaolai.cc/`这个网址

**Comment by KKISO574** [comment]
我也一样不知道是什么是不是服务器缓存原因

**Reply by linlinzzo** [reply]
是CDN缓存的原因，原先的CDN缓存时间为五秒，上周新换的CDN好像不刷新就一直缓存。现在已经将CDN换回来了。

**Comment by linlinzzo** [comment, marked as answer]
请您们在十分钟之后再反馈一下问题的恢复情况。

**Reply by tjsky** [reply]
经测试已经可以正常跳转了。辛苦了
